.gitignore

```diff
@@ -58,11 +58,21 @@ jspm_packages/
 # OS X temporary files
 .DS_Store
 
+# IntelliJ IDEA project files; if you want to commit IntelliJ settings, this recipe may be helpful:
+# https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
+.idea/
+*.iml
```

Comment:
This should be added to the `rush init` template

Reply:
😂 Didn't realize this was the TSDoc repo -- I thought it was the Rush Stack repo.

Reply:
This is from the `rush init` template :)